docs/application/dotnet/guides/security/requesting-permissions.md

```diff
@@ -141,6 +141,94 @@ To check whether an application has permission to use a privilege, and to reques
 
     If the decision is definitive, any subsequent `RequestPermission()` calls result in an immediate response with an appropriate result: `AllowForever` or `DenyForever`. However, the user can change the status of privacy-related privileges later by modifying the privacy settings on the device. For this reason, the application must always check the status of privacy-related privileges before using protected functionality.
 
+## Requesting Multiple Permissions
+
+This section describes how to check and request multiple privileges in a single API call.
+
+> **Note**
+>
+> Multiple privileges in single API call are supported from Tizen 5.5.
```

Comment:
SUggestion:
Rephrase as:
Multiple privileges in a single API call are supported from Tizen 5.5.

Please verify and update.

Reply:
@Reni-90 Regarding https://github.com/Samsung/tizen-docs/pull/589#discussion_r260610347, how about using "since Tizen 5.5"?

Reply:
@mijin85cho 
I will use @Reni-90 suggestion.